### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abrauchli**
* Provide details on where to find drivers
* Refactor the types section
* Other minor changes for improved readability

### CODE REVIEW

**Comment by abrauchli** [comment]
Rendered preview: https://github.com/Sensirion/embedded-common/blob/7ca4e29f098827bce420401d2083b8e2272d6a4f/README.md

File: README.md [hunk 1]
```diff
@@ -1,23 +1,12 @@
 # embedded-common
-This repository contains the Sensirion i2c abstraction layer and commonly shared
-code. It is a dependency of the Sensirion embedded drivers and it should not be
-necessary to clone it independently, as we recommend to download the repository
-recursively from the respective projects.
+This repository contains the Sensirion i2c hardware abstraction layer and
```
Comment (rnestler) [inline README.md:2]: s/i2c/I2C/

**Review by rnestler — APPROVED** [review]
Other than the small nitpick it LGTM.

File: README.md (hunk 1 above)
Comment (abrauchli) [reply README.md:2]: I wanted to change it, but it's consistent all over..

**Pull request merged**